fastlane/Fastfile

```diff
@@ -233,6 +233,21 @@ lane :test_release do
   )
 end
 
+desc "Runs tests in Release config on macOS"
+lane :test_release_macos do
+
+  setCIEnvironmentVariable("../Tests/StreamChatTests/StreamChatTestPlan.xctestplan")
+
+  scan(
+    project: "StreamChat.xcodeproj",
+    scheme: "StreamChat",
+    testplan: "StreamChatTestPlan",
+    configuration: "ReleaseTests",
+    clean: true,
+    destination: 'platform=macOS,arch=x86_64' # will select first from macOS and macOS Catalyst. Let's hope it always will be macOS
```

Comment:
Pity that fastlane do not support macOS proactively.
https://github.com/fastlane/fastlane/pull/9032